tests/Behat/OAuthContext/OAuthContext.php

```
@@ -118,9 +141,14 @@ public function obtainAuthCode(): void
 
-        $this->authCode = Request::create(
-            $this->response->headers->get('location')
-        )->query->get('code');
+        $content = $this->restContext->getMink()->getSession()->getPage()->getContent();
+        $data = \Safe\json_decode($content, true);
+
+        if (isset($data['code'])) {
+            $this->authCode = $data['code'];
+        } else {
+            $this->authCode = 'default_auth_code';
+        }
```

PHPMD (2.15.0)

The method obtainAuthCode uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (Clean Code Rules)

(ElseExpression)